src/aap_eda/services/project/git.py

```diff
@@ -150,6 +150,7 @@ def clone(
             cmd.extend(["--depth", str(depth)])
         cmd.extend([final_url, os.fspath(path)])
         logger.info("Cloning repository: %s", url)
+        logger.warning(f"JBS: Cloning command: {cmd}")
```

Comment:
If the use of the ssh key at import is out of the scope of this PR and it will be implemented later, I think this line is not needed at all.